public/data/ShlokMohanty.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Shlok Mohanty",
+  "bio": "Member of EddieHub",
```

Comment:
Maybe describe a bit of yourself here instead of just `Member of EddieHub`?

Reply:
This would be useful 👍 , but can come in another PR